Resources/doc/index.rst

```diff
@@ -0,0 +1,45 @@
+Installation
+============
+
+Step 1: Download the Bundle
+---------------------------
+
+Open a command console, enter your project directory and execute the
+following command to download the latest stable version of this bundle:
+
+.. code-block:: bash
+
+    $ composer require symfony/monolog-bundle "~2.7"
+
+This command requires you to have Composer installed globally, as explained
+in the `installation chapter`_ of the Composer documentation.
+
+Step 2: Enable the Bundle
+-------------------------
+
+Then, enable the bundle by adding the following line in the ``app/AppKernel.php``
+file of your project:
```

Comment:
> by adding the following line

Which line? The code example has more than a single line. 

Would be better to assume the user knows what to look for in the code example and say "Then enable the bundle by adding it to a list of registered bundles in the ``app/AppKernel.php`` file of your project:"